lib/provider.js

```diff
@@ -4,6 +4,12 @@ var contrast = require('contrast')
 var solid = JSON.parse(fs.readFileSync(__dirname + '/../solid-classes.json', 'utf-8'))
 solidClasses = solid.suggestions.classes
 solidVars = solid.suggestions.variables
+var selectors = [
+  '.html .meta.tag.any .string.quoted',
+  '.source.css.scss .meta.property-value.scss',
+  '.source.js.jsx .tag.open .string.quoted',
```

Comment:
TSX still works, but seems JSX was broken. here are two scope examples of some JSX from inside the className attribute string:

at root scope of file
```
source.js.jsx
meta.tag.jsx
string.quoted.double.js
punctuation.definition.string.end.jsx
```

inside of a class function:
```
source.js.jsx
meta.class.body.js
meta.group.braces.curly
meta.tag.jsx
string.quoted.double.js
punctuation.definition.string.end.jsx
```